go.mod

```diff
@@ -44,13 +44,13 @@ require (
 	github.com/lightninglabs/neutrino v0.13.2
 	github.com/lightninglabs/protobuf-hex-display v1.4.3-hex-display
 	github.com/lightningnetwork/lightning-onion v1.0.2-0.20220211021909-bb84a1ccb0c5
-	github.com/lightningnetwork/lnd/cert v1.1.0
+	github.com/lightningnetwork/lnd/cert v1.1.1
 	github.com/lightningnetwork/lnd/clock v1.1.0
 	github.com/lightningnetwork/lnd/healthcheck v1.2.0
 	github.com/lightningnetwork/lnd/kvdb v1.3.0
 	github.com/lightningnetwork/lnd/queue v1.1.0
 	github.com/lightningnetwork/lnd/ticker v1.1.0
-	github.com/lightningnetwork/lnd/tlv v1.0.1
```

Comment:
If we remove the replace directives, then we don't need to modify this, and instead do it in another PR after allt he new tags are pushed.